README.md

```diff
@@ -1,2 +1,5 @@
 # chat-app-in-django
 Simple chat application using django web framework.
+
+For the related things of this project, you can refer to this lucidchart.
+ - [lucidchart link](https://lucid.app/lucidchart/invitations/accept/3f78c404-d7a9-42f4-95a0-debab51e991f)
```

Comment:
Okay, so we will use this lucidchart to manage the related things for our project.